include/rocksdb/cloud/cloud_env_options.h

```diff
@@ -344,6 +351,7 @@ class CloudEnvOptions {
       int _number_objects_listed_in_one_iteration = 5000,
       int _constant_sst_file_size_in_sst_file_manager = -1,
       bool _skip_cloud_files_in_getchildren = false,
+      bool _use_direct_io_for_cloud_download = false,
```

Comment:
If it is recommended that this should be set to true, why not default it to true?